backend/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
```

Comment:
I'd suggest to use gradle's version catalog to keep track of all the dependencies and plugin versions, also because it's the only way to have dependabot work with gradle.
ref. https://docs.gradle.org/current/userguide/version_catalogs.html